model/manifest_test.go

```diff
@@ -5,23 +5,23 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestManifestFileIsInvalid(t *testing.T) {
 	t.Parallel()
 
 	_, err := GetManifest(strings.NewReader("123123 123123"))
 
-	if assert.Error(t, err) {
-		assert.Equal(t, "yaml: unmarshal errors:\n  line 1: cannot unmarshal !!str `123123 ...` into model.Manifest", err.Error())
-	}
+	require.Error(t, err)
+	assert.Equal(t, "yaml: unmarshal errors:\n  line 1: cannot unmarshal !!str `123123 ...` into model.Manifest", err.Error())
```

Comment:
Just use `assert.EqualError()` to cover both lines :)

Same with the other pair.